rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -62,20 +81,97 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.CoreMatchers.nullValue;
-import static org.hamcrest.Matchers.*;
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.fail;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
 class BridgeSerializationUtilsTest {
 
+    private static final NetworkParameters NETWORK_PARAMETERS = NetworkParameters.fromID(NetworkParameters.ID_MAINNET);
+    private static final Address ADDRESS = Address.fromBase58(NETWORK_PARAMETERS, "16zJJhTZWB1txoisGjEmhtHQam4sikpTd2");
+    private static final Address OTHER_ADDRESS = Address.fromBase58(NETWORK_PARAMETERS, "3NWYFKUegmqAjoW8aMhSLNVW2jy4MiA2eS");
+
+    @Test
+    void serializeAndDeserializeRskTxWaitingForSignatures_whenValidData_shouldReturnEqualResults() {
+        // Arrange
+        BtcTransaction prevTx = new BtcTransaction(NETWORK_PARAMETERS);
+        prevTx.addOutput(Coin.FIFTY_COINS, ADDRESS);
+
+        Keccak256 rskTxHash = createHash3(1);
+        BtcTransaction tx = new BtcTransaction(NETWORK_PARAMETERS);
```

Comment:
Let's use meaningful names for the variables.

For example:
- prevTx -> fundTx
- tx -> pegoutTx
- rskTxHash -> pegoutCreationRskTxHash

Reply:
Updated, let me know how it looks now